### ISSUE TITLE
[Bug]: Start mo time out

### ISSUE THREAD

**Issue by badboynt1** [matrixorigin/matrixone#14678]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch Name

main

### Commit ID

lastest

### Other Environment Information

```Markdown
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

bvt failed in https://github.com/matrixorigin/matrixone/actions/runs/8043747958/job/21966336462?pr=14676
start mo fail, and succeed after retry
![image](https://github.com/matrixorigin/matrixone/assets/112734932/4afab1c6-734f-49e5-b95f-80d3fa5489b7)

### Steps to Reproduce

```Markdown
see https://github.com/matrixorigin/matrixone/actions/runs/8043747958/job/21966336462?pr=14676
```

**Comment by jensenojs** [comment]
@guguducken  杰哥麻烦看看需不需要跳高一些启动的时限?

**Comment by guguducken** [comment]
实际测试main只要30s就能够拉起来，并且其是偶发，故不应该拉长启动时间限制，应该找内核同学看看

**Comment by jensenojs** [comment]
@Ariznawlll  确认这个问题复现概率不低, 但这俩jobs的mo内部日志已经拿不到了

https://github.com/matrixorigin/matrixone/actions/runs/8077322204?pr=14725

**Comment by jensenojs** [comment]
与 https://github.com/matrixorigin/MO-Cloud/issues/2264 可能有关系, 等它修好之后再观察一段时间吧, 辛苦喜亮哥一起跟踪. @qingxinhome

**Comment by qingxinhome** [comment]
处理中

**Comment by qingxinhome** [comment]
The issue has been resolved and PR has been submitted. Please verify and test it @badboynt1 @jensenojs @guguducken @aressu1985 @daviszhen

**Comment by aressu1985** [comment]
fixed: 275af9cc9a4d113bd5ddcb5381eb9a9d15e0dd87
<img width="1439" alt="image" src="https://github.com/matrixorigin/matrixone/assets/47846308/4b92a541-04b5-4de4-9b2b-037c2072e66e">
